release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -156,41 +156,41 @@ In the following tables, features are marked with the following statuses:
 
 |====
 
-[discrete]
-=== Monitoring deprecated and removed features
+//[discrete]
+//=== Monitoring deprecated and removed features
 
-.Monitoring deprecated and removed tracker
-[cols="4,1,1,1",options="header"]
-|====
-|Feature |4.12 |4.13 |4.14
+//.Monitoring deprecated and removed tracker
+//[cols="4,1,1,1",options="header"]
+//|====
+//|Feature |4.12 |4.13 |4.14
 
-|====
+//|====
 
 [discrete]
 === Installation deprecated and removed features
 
 .Installation deprecated and removed tracker
 [cols="4,1,1,1",options="header"]
 |====
-|Feature |4.11 |4.13 |4.14
+|Feature |4.12 |4.13 |4.14
 
 |vSphere 7.0 Update 1 or earlier
 |General Availability
 |Removed ^[1]^
-|
+|Removed
 
 |VMware ESXi 7.0 Update 1 or earlier
 |General Availability
 |Removed ^[1]^
-|
+|Removed
 
 |CoreDNS wildcard queries for the `cluster.local` domain
-|General Availability
+|Deprecated
 |Deprecated
 |Deprecated
```

Comment:
Based on the unstated rule of two cycles of deprecation before we remove something, I would need to check with engineers that this feature is now removed.

Reply:
![Screenshot from 2023-06-28 15-02-35](https://github.com/openshift/openshift-docs/assets/57954076/68f97eeb-1d05-4b9c-9646-f1b5832e13b8)